### PR TITLE
Add pt_BR localization

### PR DESCRIPTION
**Pull request by DatCrazyDelphox**
Now, that paru is accepting them, i created one for this language

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+msgid "Repo Make"
+msgstr "Pacotes Make no repositório"
+
+#: src/install.rs:1138
+msgid "Aur Make"
+msgstr "Pacotes Make no AUR"
+
+#: src/install.rs:1240
+msgid "fetching devel info..."
+msgstr "buscando informações de compilação..."
```
Comment (Morganamilo) [inline po/pt_BR.po:504]: Ideally these will be kept as short as possible. Repository still starts with Repo. Does it make sense in the language to use Repo and Repo Make?

Comment (DatCrazyDelphox) [reply po/pt_BR.po:504]: Not really, I just followed pacman's own localization, but I'll fix it, no problem

Comment (DatCrazyDelphox) [reply po/pt_BR.po:504]: Just some parts that really need to be called 'repositórios', because of legibility

Comment (Morganamilo) [reply po/pt_BR.po:504]: It's fine if they are. I was just asking, it's not like I know the language.

**Comment by not-a-dev-stein** [comment]
Damn, just a couple of hours faster than me. Just finished my translation, but looking at yours there's a lot better with your version.

File: po/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+msgid "Missing ABS packages "
+msgstr "Pacotes ABS faltando "
+
+#: src/download.rs:250 src/download.rs:276 src/download.rs:530
+#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
+#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
+#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
+#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+msgid "failed to run:"
+msgstr "erro ao rodar:"
```
Comment (Morganamilo) [inline po/pt_BR.po:207]: See https://github.com/Morganamilo/paru/pull/435#discussion_r660053767

Comment (DatCrazyDelphox) [reply po/pt_BR.po]: Fixed that

**Comment by Morganamilo** [comment]
Looks good now thanks.

**Pull request merged**